argcomplete/__init__.py

```diff
@@ -245,7 +237,7 @@ def __call__(self, argument_parser, always_complete_options=True, exit_method=os
             completions = [dfs.join((key, display_completions.get(key) or "")) for key in completions]
 
         debug("\nReturning completions:", completions)
-        output_stream.write(ifs.join(completions).encode(sys_encoding))
+        output_stream.write(ifs.join(completions).encode())
```

Comment:
Alternatively, if we open `output_stream` in text mode rather than binary mode, its default encoding will already match `locale.getpreferredencoding()` and we can write strings to it directly.

Reply:
Thanks for the review! I think now that we're off of Python 2, it makes sense to open output_stream in text mode. I'll update that.